_includes/footer.html

```diff
@@ -0,0 +1 @@
+<footer>Footer</footer>
```

Comment:
```suggestion
<footer>
    <p>Footer</p>
</footer>
```
Your HTML is completely valid I believe but we usually place text with a surrounding text tag. For example a `<p>`, `<h1>`, `<li>` and so on. See [here](https://stackoverflow.com/a/43040453/1480728)